### ISSUE TITLE
Suppresed arel exception, wrong sql

### ISSUE THREAD

**Issue by printercu** [rgeo/activerecord-postgis-adapter#207]
Hi!

I tried to access json fields with `Arel::Nodes::InfixOperation.new('->>', arel_table[:attr], 'field')`. With `adapter: postgis` it doesn't raise exception but produces wrong sql: `"table"."attr" ->> ST_GeomFromEWKT('field') =`. Accessing json arrays with same operation (int instead of string) worked fine.

With `adapter: postgresql` it raises exception from arel `RuntimeError: unsupported: String`. I had an error in my code, and then fixed it by replacing string with `Arel::Nodes.build_quoted('field')` (sql became `"table"."attr" ->> 'field' =`).

I think it'll be better to bypass this exception (if it's possible of course).

**Comment by kitsunde** [comment]
I'm seeing the same issue in 2020 on the latest `activerecord-postgis-adapter`, somehow `ST_GeomFromEWKT` is polluting very basic Arel operations.

```
Loading development environment (Rails 6.0.3.2)
Available Tenants: ["public", "central_cg"]
Use `st 'tenant'` to switch tenants & `tenant_list` to see list
[1] [development][public] pry(main)> Arel::Nodes::Count.new(['*']).to_sql
=> "COUNT(ST_GeomFromEWKT('*'))"
```

**Comment by kitsunde** [comment]
Right this is a duplicate of #183

**Comment by keithdoggett** [comment]
closed by #325
